bin/makepart

```diff
@@ -29,6 +29,7 @@ if ! grep _secureboot <<<"$features" > /dev/null; then
 	touch "$rootfs_work/boot/efi/loader/loader.conf"
 	touch "$rootfs_work/boot/efi/loader/entries.srel"
 	chroot "$rootfs_work" /usr/bin/env -i SYSTEMD_ESP_PATH="/boot/efi" bootctl --no-variables --make-machine-id-directory=no install
+	echo type1 > "$rootfs_work/boot/efi/loader/entries.srel"
```

Comment:
Little detail: Missing new Line after "type1"

E.g.: `echo -e "type1\n" > "$rootfs_work/boot/efi/loader/entries.srel"` 

The [systemd documentation](https://systemd.io/BOOT_LOADER_SPECIFICATION/) mentiones that 

> [...] In order to minimize confusion, a boot loader implementation may place the file /loader/entries.srel next to the /loader/entries/ directory containing the ASCII string type1 (followed by a UNIX newline). [...]

The bootctl install loader spec adds a new line ([source](
https://github.com/systemd/systemd/blob/78fa2f91d208b51f2a4878012b9558389e9a7660/src/boot/bootctl.c#L1373))